src/styles/header.scss

```diff
@@ -0,0 +1,74 @@
+@import './variables';
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: $color-white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  gap: 20px;
+}
+
+.nav {
+  margin-left: 20px;
+}
+
+.nav:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: $card-main-color;
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: $card-buy-color;
+}
+
+.nav__link.is-active {
+  color: $card-buy-color;
```

Comment:
The active link color is the same as the hover color. Consider using a different style or indicator for active links to distinguish them from hover states.